modules/ossm-uguide-routing.adoc

```diff
@@ -0,0 +1,264 @@
+
+[id="ossm-routing-lb_{context}"]
+==== Load balancing options 
+
+By default, {ProductName} uses a round-robin load balancing policy, where each service instance in the instance pool gets a request in turn. {ProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
+
+* Random: Requests are forwarded at random to instances in the pool.
+* Weighted: Requests are forwarded to instances in the pool according to a specific percentage.
+* Least requests: Requests are forwarded to instances with the least number of requests.
+
+.Destination example
+
+The following example destination rule configures three different subsets for the `my-svc` destination service, with different load balancing policies:
+
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: my-destination-rule
+spec:
+  host: my-svc
+  trafficPolicy:
+    loadBalancer:
+      simple: RANDOM
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+    trafficPolicy:
+      loadBalancer:
+        simple: ROUND_ROBIN
+  - name: v3
+    labels:
+      version: v3
+----
+
+[id="ossm-routing-gw_{context}"]
+=== Gateways
+
+You can use a gateway to manage inbound and outbound traffic for your mesh to specify which traffic you want to enter or leave the mesh. Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than sidecar Envoy proxies running alongside your service workloads.
+
+Unlike other mechanisms for controlling traffic entering your systems, such as the Kubernetes Ingress APIs, {ProductName} gateways let you use the full power and flexibility of traffic routing. The {ProductName} gateway resource can layer 4-6 load balancing properties such as ports to expose, {ProductName} TLS settings. Instead of adding application-layer traffic routing (L7) to the same API resource, you can bind a regular {ProductName} virtual service to the gateway and manage gateway traffic like any other data plane traffic in a service mesh.
+
+Gateways are primarily used to manage ingress traffic, but you can also configure egress gateways. An egress gateway lets you configure a dedicated exit node for the traffic leaving the mesh, letting you limit which services have access to external networks, or to enable secure control of egress traffic to add security to your mesh, for example. You can also use a gateway to configure a purely internal proxy.
+
+.Gateway example
+
+The following example shows a possible gateway configuration for external HTTPS ingress traffic:
+
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: ext-host-gwy
+spec:
+  selector:
+    app: my-gateway-controller
```

Comment:
I know this is just an example, but if the user copies and pastes this code, it won't work as is, because this gateway controller does not exist out of the box.

So, I suggest changing this line to: `istio: ingressgateway # use istio default controller`, as seen here: https://github.com/istio/istio/blob/release-1.4/samples/bookinfo/networking/bookinfo-gateway.yaml#L7